src/staticanalysis/bot/tests/test_reporter_phabricator.py

```diff
@@ -12,7 +12,7 @@
 
 VALID_CLANG_TIDY_MESSAGE = '''
 Code analysis found 1 defect in this patch:
```

Comment:
I would prefer it is published on this line

Reply:
Agreed, it would be better to do:

```
Code analysis found 1 defect in the diff n°XXXX
```

As phabricator itself mention `diff`

Reply:
Indeed it looks way better like this.